nityam_kumar/PostgresAssignment/src/config/validation.ts

```diff
@@ -0,0 +1,132 @@
+interface Student {
+  cl_no: number;
+  age: number;
+  st_id?: string;
+  fname: string;
+  lname: string;
+  sex?: string;
+}
+
+interface Teacher {
+  age: number;
+  fname: string;
+  lname: string;
+  teacher_id?: string;
+  sex?: string;
+}
+
+interface Mark {
+  st_id: string;
+  sub_id: string;
+  marks: number;
+  teacher_id: string;
+  cl_no: number;
+}
+
+interface Subject {
+  sub_id?: string;
+  sub_name: string;
+  teacher_id: string;
+  cl_no: number;
+}
+
+export function checkStudent(s1: Student) {
+  if (
```

Comment:
we can use Joi for validation. https://joi.dev/api/?v=17.4.0
Try it out in 1 or 2 places. No need to change in alll the places.